lib/commands/connect/restart.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+var api = require('./shared.js');
+let cli = require('heroku-cli-util');
+
+module.exports = {
+    topic: 'connect',
+    command: 'restart',
+    description: 'Restart a connection',
+    help: 'Clears errors and attempts to resume sync operations',
+    flags: [
+      {name: 'resource', char: 'r', description: 'specific connection resource name', hasValue: true}
+    ],
+    needsApp: true,
+    needsAuth: true,
+    run: cli.command(function (context, heroku) {
+      api.withUserConnections(context.auth.password, context.app, context.flags, false, heroku).then(function(connections){
+          if (connections.length > 1) {
+            console.log("Multiple connections found. Please use '-r' to specify a single connection by resource name.");
+            api.connection_info(connections);
+          } else {
+            connections.forEach(function(connection) {
+              api.request(context.auth.password, 'POST', '/api/v3/connections/' + connection.id + '/actions/restart')
+                .then(function(response) {
+                  switch(response.statusCode) {
+                    case 202:
+                      console.log(api.connection_string(connection) + ' restarted');
+                    break;
+                    case 409:
+                      console.log(response.json['message']);
```

Comment:
Might be nice to include some context around the connection that the message goes with, like is done in the case of the 202 case. Should we have any default case, and just have a generic "... received invalid response ... unable to restart ..." or something to that effect?